supertokens_python/recipe/emailpassword/recipe.py

```diff
@@ -175,6 +175,7 @@ async def handle_api_request(
         path: NormalisedURLPath,
         method: str,
         response: BaseResponse,
+        user_context: Dict[str, Any],
```

Comment:
you need to pass in this to the apis called in this.. see node pr

Reply:
Fixed.